## iterative Linear Quatratic Regulator

controller/iLQR:
  N:    &N  15
  dt:   &dt 0.1
  v_max: 1.2
  omega_max: 2.0

controller/cost: 
  N: *N
  
  # Weight Matrices
  Qf_x:     1.0
  Qf_y:     1.0
  Qf_theta: 1.0

  Q_x:      1.0
  Q_y:      1.0
  Q_theta:  1.0

  R_v:      1.0
  R_omega:  1.0

controller/dynamics:
  dt: *dt
  model: "standard"

## Odometry Params
odom:
  topic: "/terrasentia/will/odom"
  frame_id: "/terrasentia/will/odom"